stdlib/public/core/IntegerParsing.swift.gyb

```diff
@@ -80,7 +80,7 @@ internal func _parseAsciiAsUIntMax(
     else { return nil }
   // Disallow < 0.
   if hasMinus && result != 0 { return nil }
-  // Return.
+  // Valid result, return.
   return result
 }
```

Comment:
@dabrahams I can definitely see where you're coming from but let me explain myself :).

**Rationale**
The comments headings are intended for code grouping (instead of newlines) so you can quickly skim the algorithm and it's structure/flow just from those headings. The `return` here doesn't belong to the `// Disallow < 0` code. You could separate them with a newline, but a comment is a bit more consistent with the rest of the method, and while skimming the headings conveys the fact that we early-returned all invalid results.